EIPS/eip-3382.md

```diff
@@ -0,0 +1,50 @@
+---
+eip: 3382
+title: Hardcoded Block Gas Limit
+author: Philippe Castonguay (@PhABC)
+discussions-to: https://ethereum-magicians.org/t/eip-3382-hardcoded-gas-limit
+status: Draft
+type: Standards Track
+category: Core
+created: 2020-03-13
+---
+
+## Simple Summary
+
+Hardcode the block gas limit (known as *block gas target* under [EIP-1559](https://github.com/ethereum/EIPs/blob/master/EIPS/eip-1559.md)) to `12,500,000` gas per block.
+
+## Abstract
+
+Ethereum's block gas limit is currently dictated by block producers and is not enforced when validating blocks in clients. This EIP proposes to hardcode the block gas limit to `12,500,000`.
```

Comment:
Abstract should be a human readable and terse version of the specification.  The first sentence you have talks about history, which isn't appropriate for the specification/abstract.  The second sentence is a tad less clear than I think would be ideal.

```suggestion
Updates the block validation rules such that a block is invalid if the `gas_target` header field is not equal to 12,500,000.
```